pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -830,7 +830,7 @@ func (manager *GCEServiceManager) CreateRegionalDisk(
 			manager.gce.projectID, manager.gce.region, diskToCreateAlpha).Do()
 	}
 
-	return nil, fmt.Errorf("The regional PD feature is only available via the GCE Alpha API. Enable \"GCEDiskAlphaAPI\" in the list of \"alpha-features\" in \"gce.conf\" to use the feature.")
+	return nil, fmt.Errorf("The regional PD feature is only available via the GCE Alpha API. Enable %q in the list of \"alpha-features\" in \"gce.conf\" to use the feature.", GCEDiskAlphaFeatureGate)
```

Comment:
Do you want to rename GCEDiskAlphaFeatureGate to be consistent with the networking feature that's also in the enum?

Reply:
Renamed to `AlphaFeatureGCEDisk` to align with the networking feature enum

Reply:
This was also done in https://github.com/kubernetes/kubernetes/pull/51067, rebasing.